### DISCUSSION TITLE
vweb file upload example -- change the example so the file is saved instead of displayed raw.

### DISCUSSION THREAD

**Discussion by reyvaleza** [vlang/v#14609] [Ideas]
Will you please make the file upload example in vweb save the file instead of simply displaying the raw data? Please. Thanks!

**Comment by Tanguy-SALMON** [comment]
Here an example: #9615

**Comment by reyvaleza** [comment]
Thank you!

**Comment by enghitalo** [comment]
You can also see https://github.com/vlang/v/tree/master/vlib/vweb there are some examples in there

**Reply by reyvaleza** [reply]
Thanks!
